cmd/src/server.rs

```diff
@@ -472,6 +473,9 @@ impl TiKVServer {
             Box::new(gc_worker.get_config_manager()),
         );
 
+        let (txn_extra_tx, txn_extra_rx) = mpsc::unbounded();
+        self.engines.as_mut().unwrap().engine.txn_extra_tx = Some(txn_extra_tx);
```

Comment:
Could we set `txn_extra_tx ` when create `engines`?

Reply:
But it needs `init_engines` to return the receiver.

Reply:
Please use 'set_{{param}}' method instead of 'public param'.